### PR TITLE
fix injector v2 issues

### PR DESCRIPTION
**Pull request by Hyferion**

### CODE REVIEW

File: components/RewardsInjectorConfigurator.tsx [hunk 1]
```diff
@@ -118,9 +118,11 @@ function RewardsInjectorConfigurator({
       const gaugeTotal = amount * maxPeriods;
       const gaugeDistributed = amount * currentPeriod;
 
-      total += gaugeTotal;
-      distributed += gaugeDistributed;
-      remaining += gaugeTotal - gaugeDistributed;
+      total += gaugeTotal / 10 ** tokenDecimals;
```
Comment (Xeonus) [inline components/RewardsInjectorConfigurator.tsx:121]: This is great thx!

File: components/tables/RewardsInjectorTable.tsx [hunk 2]
```diff
@@ -181,14 +203,18 @@ export const RewardsInjectorTable: React.FC<RewardsInjectorTableProps> = ({
...
+                <Td>{row.poolName}</Td>
+              )}
               <Td
                 isNumeric
               >{`${Number(row.amountPerPeriod).toFixed(2)} ${tokenSymbol}`}</Td>
               <Td isNumeric>{row.periodNumber}</Td>
               <Td isNumeric>{row.maxPeriods}</Td>
               <Td>{formatDate(row.lastInjectionTimeStamp)}</Td>
+              {isV2 && row.doNotStartBeforeTimestamp && (
+                <Td>{formatDatetime(row.doNotStartBeforeTimestamp)}</Td>
```
Comment (Xeonus) [inline components/tables/RewardsInjectorTable.tsx:216]: Looks a bit cut off on my test
image

**Review by Xeonus — CHANGES_REQUESTED** [review]

File: components/tables/RewardsInjectorTable.tsx (hunk 2 above)
Comment (Hyferion) [reply components/tables/RewardsInjectorTable.tsx:223]: fixed

File: components/RewardsInjectorConfigurator.tsx (hunk 1 above)
Comment (Xeonus) [reply components/RewardsInjectorConfigurator.tsx:121]: Looks good now

**Review by Xeonus — APPROVED** [review]
All looks good now! Thx a lot fine ser!

**Pull request merged**